src/components/MultiSelect/MultiSelect.scss

```
@@ -1,10 +1,80 @@
 @use "sass:map";
-@import "vanilla-framework";
-@include vf-base;
-@include vf-p-lists;
+
+@import "~vanilla-framework/scss/_base_forms";
```

Stylelint seems to not like the leading `_` in `_base_forms`. :slightly_frowning_face: 
In my opinion, we can safely ignore this. We are constrained by how this is named in Vanilla, so there's not much we can do about it.
```suggestion
// stylelint-disable-next-line scss/load-no-partial-leading-underscore
@import "~vanilla-framework/scss/_base_forms";
```